common/my_filters.py

```diff
@@ -35,3 +35,8 @@ def yt_start(value):
 @register.filter
 def get_item(dictionary, key):
     return dictionary.get(key)
+
+
+@register.filter
+def strftime(d, str):
```

Comment:
You should be able to use this instead.

https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#date